desc/configuration.py

```diff
@@ -1012,11 +1066,46 @@ def iota(self, new):
     @property
     def i_l(self):
         """Coefficients of iota profile (ndarray)."""
-        return self.iota.params
+        try:
+            return self.iota.params
+        except AttributeError:
+            return None
```

Comment:
should this be an empty ndarray instead of None? Are there places we expect it to be a certain type?

Reply:
You're right. The changes I'll push for your [other comment](https://github.com/PlasmaControl/DESC/pull/251#discussion_r966294580) expect it to be an empty ndarray too.